db/migrate/20241119142129_create_instructeurs_procedures.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreateInstructeursProcedures < ActiveRecord::Migration[7.0]
+  def change
+    create_table :instructeurs_procedures do |t|
+      t.references :instructeur, null: false, foreign_key: true
+      t.references :procedure, null: false, foreign_key: true
+      t.float :position
```

Comment:
```suggestion
      t.float :integer
```

a confirmer ac @tchak ; mais je pense qu'on peut opt-out de la solution parfaite car on fait du bulk au final (pour des raisons d'UX... cad: on souhaite pouvoir annuler les chgmt en cours de route, donc on fait pas du changement de position unitaire, on change tout d'un coup)

PS: je pense que tu peux rejouer la migration facilement via rails db:migrate:redo VERSION=20241119142129